source/user/installation_guide.rst

```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
 All the data and configuration needed to run a simulation are located in the `plug`_
 repository, along with usage examples.
 
+Natively
+--------
+
+The framework can be used natively after some initial setup. This first requires the following:
+
++ install julia 1.5 and create an environment as described in `REISE.jl`_
```

Comment:
Just to be consistent with the rest of the doc, should these bullet points be capitalized?